src/appComponents/twitterwall/singletweet.js

```diff
@@ -55,7 +55,7 @@ function tweet($timeout,$rootScope) {
     }
     function changeTweet() {
 
-      if (scope.tweetNumber !=== 90) {
+      if (scope.tweetNumber !== 90) {
             scope.tweetNumber = scope.tweetNumber + 1;
             scope.selectedid  =scope.array[scope.tweetNumber].id_str;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'id_str' is not in camel case.](https://www.codacy.com/app/triplez/open-event-webapp/file/2532205002/issues/source?bid=3233857&fileBranchId=3284227#l60)